src/util/sql/transform-placementextent.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE ${migrationSchema:name}.evaka_service_need (
     option_id UUID,
     placement_id UUID NOT NULL REFERENCES ${migrationSchema:name}.evaka_placement,
     start_date DATE NOT NULL,
-    end_date DATE NOT NULL,
```

Comment:
Doesn't seem right, service_need#end_date is not nullable.

Reply:
Changed back to non-nullable, but infinity is now default due to parser only producing null values for the source data.